### PR TITLE
Conformance2.0 correct intent cases

### PR DESCRIPTION
**Pull request by kriswest**
Numerous corrections to 2.0 test cases - particularly in the intent test cases.

### CODE REVIEW

File: toolbox/fdc3-conformance/FDC3-2.0-Conformance-Test-Cases.md [hunk 1]
```diff
@@ -275,55 +278,58 @@ Finally, please note that this is a larger set of apps than were required for 1.
...
+  - Then in the first step, use `fdc3.raiseIntent("aTestingIntent", testContextX, appIdentifier)` to target the running instance of app A. 
+  - Confirm that the intent is delivered to the correct instance and that another instance is NOT started. Otherwise, as above.
 - `2.0-RaiseIntentTargetedInstanceResolveFindInstances`: Repeat the above test, but:
-  - Before the first step, use `let appIdentifier = await fdc3.open({appId: "<A's appId>"})` to start A and retrieve its `AppIdentifier` with instance details.
-  - Then in the first step, use `fdc3.raiseIntent(‘aTestingIntent’, testContextX, appIdentifier)` to target the running instance of app A, otherwise, as above.
+  - Before the first step, use `let appIdentifier = await fdc3.open({appId: "<A's appId>"})` to start A.
+  - Then use `const instances = await fdc3.findInstances({appId: "<A's appId>"})` to retrieve a list of instances of app A. Confirm that only one is present and retrieve its `AppIdentifier`, confirming that it contains and `instanceId` field that matches that returned by the `fdc3.open` call.
```
Comment (Joe-Dunleavy) [inline toolbox/fdc3-conformance/FDC3-2.0-Conformance-Test-Cases.md:295]: minor typo. 'and' should be 'an' -> '...confirming that it contains and `instanceId`...'

**Review by Joe-Dunleavy — APPROVED** [review]
just a small typo that needs fixing

File: toolbox/fdc3-conformance/FDC3-2.0-Conformance-Test-Cases.md (hunk 1 above)
Comment (kriswest) [inline toolbox/fdc3-conformance/FDC3-2.0-Conformance-Test-Cases.md:295]:
```suggestion
  - Then use `const instances = await fdc3.findInstances({appId: "<A's appId>"})` to retrieve a list of instances of app A. Confirm that only one is present and retrieve its `AppIdentifier`, confirming that it contains an `instanceId` field that matches that returned by the `fdc3.open` call.
```

Comment (kriswest) [reply toolbox/fdc3-conformance/FDC3-2.0-Conformance-Test-Cases.md]: :+1 fixed

**Pull request merged**